src/components/__tests__/ListItem.spec.tsx

```diff
@@ -79,4 +113,16 @@ describe('ListItem', () => {
     expect(onPress).not.toHaveBeenCalled()
     expect(arrowIcon.props.fill).toBe(COLORS.primary)
   })
+    it('should not call on press callback if scrolling with locked exercise', async () => {
```

Comment:
Please remove this test, it also does not test anything at all, which is not tested by other tests in this file.